debezium-connector-jdbc/pom.xml

```diff
@@ -202,6 +202,7 @@
         <dependency>
             <groupId>com.oracle.database.jdbc</groupId>
             <artifactId>ojdbc11</artifactId>
+            <version>23.6.0.24.10</version>
```

Comment:
Why it is necessary to override the version here?

Reply:
Sorry, my bad! I removed the version tag.

Reply:
@jpechane 
I also added config variable `ORACLE_INFINITY_CONVERTER_ENABLE` for the feature,
but have trouble accessing it from [convertInfinityTimestampValue()](https://github.com/dongwook-chan/debezium/blob/83b6c01558ad883633c443c4b42f248bd8ad53ec/debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/dialect/mysql/ZonedTimestampType.java#L40-L54). Could you give some advice on this? And I would like some advice on running integration test too.